dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/call/AbstractServerCallListener.java

```diff
@@ -46,7 +47,7 @@ public AbstractServerCallListener(RpcInvocation invocation, Invoker<?> invoker,
         this.responseObserver = responseObserver;
     }
 
-    public void invoke() {
+    public void invoke(TripleFlowControlFrame tripleFlowControlBean) {
```

Comment:
`call` may only need a common `data` class as param, no need to know flow control

Reply:
change to object